CHANGELOG.md

```diff
@@ -1,2 +1,6 @@
 - Fixed issue where MCP server didn't detect if iOS app uses Crashlytics in projects that use `project.pbxproj` (#9515)
 - Add logic to synchronize v2 scheduled function timeout with Cloud Schduler's attempt deadline (#9544)
+- Updated Data Connect emulator to v2.17.3:
+  - Fixed Swift codegen: Include FirebaseCore import in the connector keys file.
+  - Fixed a bug where debug details of Internal errors were swallowed.
+  - Added support to execute requests in ConnectorStreamService.Connect bidirectional stream.
```

Comment:
Let's leave out `ConnectorStreamService.Connect` in the changelog. Realtime isn't ready to use until SDK is integrated.